sys/include/net/gnrc/mac/mac.h

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (C) 2015 Daniel Krebs
+ *               2016 INRIA
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    net_gnrc_mac  A MAC module for providing common MAC parameters and helper functions
+ * @ingroup     net
```

Comment:
```
@ingroup     net_gnrc
```

and also consider, that in `@defgroup  net_gnrc_mac X` the X is the human-readable *name* of the module, not the description. So please use some shorter title format (e.g. "Common MAC module"?)